### PR TITLE
core(legacy-javascript): use prescriptive language in title

### PR DESCRIPTION
**Pull request by connorjclark**

### CODE REVIEW

File: lighthouse-core/audits/legacy-javascript.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const i18n = require('../lib/i18n/i18n.js');
 
 const UIStrings = {
   /** Title of a Lighthouse audit that tells the user about legacy polyfills and transforms used on the page. This is displayed in a list of audit titles that Lighthouse generates. */
-  title: 'Legacy JavaScript',
+  title: 'Avoid legacy JavaScript',
```
Comment (paulirish) [inline lighthouse-core/audits/legacy-javascript.js:27]: how about

> Avoid publishing legacy JavaScript to modern browsers

Comment (connorjclark) [reply lighthouse-core/audits/legacy-javascript.js:27]: publishing => serving?

Comment (paulirish) [reply lighthouse-core/audits/legacy-javascript.js:27]: sg

**Comment by connorjclark** [comment]
@paulirish review?

**Pull request merged**